src/Deprecated80/RPackage.extension.st

```diff
@@ -0,0 +1,8 @@
+Extension { #name : #RPackage }
+
+{ #category : #'*Deprecated80' }
+RPackage >> categoryName [
+	self
+		deprecated: 'This method will be removed. Clients should call name directly'.
```

Comment:
We could have an automatic rewrite for this one.

Reply:
yes, that would be good.

Reply:
```
transformWith: '`@receiver categoryName' -> '`@receiver name'
```